scripts/x-date-pickers-pro.exports.json

```diff
@@ -4,10 +4,7 @@
   { "name": "ArrowDropDownIcon", "kind": "Variable" },
   { "name": "ArrowLeftIcon", "kind": "Variable" },
   { "name": "ArrowRightIcon", "kind": "Variable" },
-  { "name": "BaseMultiInputFieldProps", "kind": "Interface" },
```

Comment:
Should we list these in the changelog of the PR as well? 🤔

Reply:
Yep, I'm waiting for the alpha 0 to be released to have a clearer vision of how we want to phrase those changes but I'll add them in the PR description :+1: